Show link as simple text (#39)
wyeworks/boom: README.md

BoomNotifier is released under the terms of the [MIT License](https://github.com/wyeworks/boom/blob/master/LICENSE).

## Credits

> The authors of this project are [Ignacio](https://github.com/iaguirre88) and [Jorge](https://github.com/jmbejar). It is sponsored and maintained by [Wyeworks](https://www.wyeworks.com).

Reviewer: What do you think about getting rid of the author section?

```suggestion
This project is sponsored by [Wyeworks](https://www.wyeworks.com) and maintained by [Ignacio](https://github.com/iaguirre88), [Jorge](https://github.com/jmbejar) and [Daniel](https://github.com/eldano)
```

Author: Maybe something to discuss in another PR as this just aims to fix the display of the img. Are you ok with this?

Reviewer: Absolutely
